### DISCUSSION TITLE
(Suggestion) Editing GUI for making Piano songs

### DISCUSSION THREAD

**Discussion by JohnSmith2007** [Monika-After-Story/MonikaModDev#7977] [general - piano notes]
I'm wanting to try my hand at making a custom piano song, however i can't get my head around the JSON formatting. So i propose that somebody should make a note editing application. Something similar to editors like REAPER & Moonscraper, both of which are mainly used for making custom content for Rock Band & Guitar Hero.

**Comment by ThePotatoGuy** [comment]
moving this to discussion. We probably won't put any focus on making one ourselves right now, but maybe someone in the community will make one.
